docs/INSTALLATION_GOOGLE.md

```diff
@@ -1,6 +1,8 @@
 # How to install the Open Event Web App Generator on Google Cloud
 This documentation provides instructions to deploy the Open Event Web App on Google Cloud using the App Engine flexible environment.
 
+[![ASCII Cast of Complete Process](https://asciinema.org/a/156500.png)](https://asciinema.org/a/156500)

```

Comment:
I do not think adding an image instead of link is helpful. Please use the link only

Reply:
You sure ?
In other setup docs, excessive uses of images are used i.e. heroku

I'm just asking, will do after your reply. :blush:

Reply:
@0x48piraj Yes, we do not need to add a screenshot of the video to link it. Please use the link only. If possible please improve the quality of the docs.